### PR TITLE
Fix the search of descriptorSets in vkUpdateDescriptorSets for copy operation.

### PR DESCRIPTION
**Pull request by yalcinmelihyasin**
b/140988159

### CODE REVIEW

File: gapis/api/vulkan/api/descriptor.api [hunk 1]
```diff
@@ -643,13 +643,13 @@ cmd void vkUpdateDescriptorSets(
   for _ , _ , c in copies {
     srcSet := DescriptorSets[c.SrcSet]
     dstSet := DescriptorSets[c.DstSet]
-    srcBinding := switch (c.SrcBinding in srcSet.Bindings) {
+    srcBinding := switch srcSet.Bindings[c.SrcBinding] == null {
       case false:
```
Comment (hevrard) [inline gapis/api/vulkan/api/descriptor.api]: For readability, I would prefer keeping `(c.SrcBinding in srcSet.Bindings)` and inverse the true/false of the `case` expressions.

You would also need to mirror this style by changing the condition on line 576

**Comment by yalcinmelihyasin** [comment]
I am not sure about this fix. Please comment on it.

File: gapis/api/vulkan/api/descriptor.api (hunk 1 above)
Comment (AWoloszyn) [reply gapis/api/vulkan/api/descriptor.api]: `c.SrcBinding in srcSet.Bindings` is NOT semantically equivalent to `srcSet.Bindings[c.SrcBinding] == null`. 
The second being correct in more cases.

Comment (hevrard) [reply gapis/api/vulkan/api/descriptor.api]: Oh ok, so we have to keep the `== null` test.

**Comment by yalcinmelihyasin** [comment]
Forced push due to flaky(possibly) test.

**Pull request merged**